### ISSUE TITLE
Refresh of active plugin on undo/redo

### ISSUE THREAD

**Issue by danyill** [openscd/open-scd-core#93]
Currently in an open-scd-core plugin, as far as I can tell undo/redo buttons do not request an update on the currently in use plugin. Or -- perhaps the `editCount` is not being correctly modified.

The undo/redo action is happening correctly but the visual display does not appear to be updated.

I have ensured that my plugin has properties on it which should effect a change, so the plugin begins:

```js
@property({ attribute: false })
doc!: XMLDocument;

@property() docName!: string;

@property() editCount!: number;
```

It looks as though `editCount` is `undefined`:

![image](https://user-images.githubusercontent.com/674783/232181225-7db98dd3-b5e8-4704-a3b5-d69d0f79e66a.png)

Even though edits have been made:

![image](https://user-images.githubusercontent.com/674783/232181234-2d1f7031-2335-4e71-ad60-25168f0e5841.png)

Happy to provide additional information.

**Comment by danyill** [comment]
I suspect this was resolved in d3b745a and may not be an issue with a new core release.

**Comment by danyill** [comment]
I can confirm that this is not an issue when working off the main branch, so I will close this issue.
